### DISCUSSION TITLE
Error Setting Time Zone

### DISCUSSION THREAD

**Discussion by dalogue1** [Nachtzuster/BirdNET-Pi#105] [General]
I tried to set the Time Zone in the BirdNET-PI and since then I get the following error in the log:

Any ideas what I have done wrong?  (I am running a newly created RPi 5 BirdNET-Pi)

I can get rid of the error by setting the Time Zone to Europe/London.  (Is it because I have not set the Time Zone correctly on the base Linux system. (I did not touch it after installation.) 

3:47:34---[server][INFO] READING AUDIO DATA...
13:47:34---[server][INFO] READING DONE! READ 5 CHUNKS.
13:47:34---[server][INFO] ANALYZING AUDIO...
13:47:35---[server][INFO] DONE! Time 0.52 SECONDS
13:47:35---[server][INFO] 0.0;3.0-('Vireo olivaceus_Red-eyed Vireo', 0.14190216)
13:47:35---[server][INFO] 3.0;6.0-('Sittasomus griseicapillus_Olivaceous Woodcreeper', 0.21206059)
13:47:35---[server][INFO] 6.0;9.0-('Polioptila dumicola_Masked Gnatcatcher', 0.34682637)
13:47:35---[server][INFO] 9.0;12.0-('Corvus brachyrhynchos_American Crow', 0.09433588)
13:47:35---[server][INFO] 12.0;15.0-('Charadrius vociferus_Killdeer', 0.118307024)
13:47:35---[birdnet_analysis][ERROR] Unexpected error:
13:47:35---Traceback (most recent call last):
13:47:35---  File "/usr/local/bin/birdnet_analysis.py", line 111, in handle_reporting_queue
13:47:35---    update_json_file(file, detections)
13:47:35---  File "BirdNET-Pi/scripts/utils/reporting.py", line 127, in update_json_file
13:47:35---    write_to_json_file(file, detections)
13:47:35---  File "BirdNET-Pi/scripts/utils/reporting.py", line 134, in write_to_json_file
13:47:35---    dets = {'file_name': os.path.basename(json_file), 'timestamp': file.iso8601, 'delay': conf['RECORDING_LENGTH'],
13:47:35---                                                                   ^^^^^^^^^^^^
13:47:35---  File "BirdNET-Pi/scripts/utils/helpers.py", line 81, in iso8601
13:47:35---    current_iso8601 = self.file_date.astimezone(get_localzone()).isoformat()
13:47:35---                                                ^^^^^^^^^^^^^^^
13:47:35---  File "BirdNET-Pi/birdnet/lib/python3.11/site-packages/tzlocal/unix.py", line 219, in get_localzone
13:47:35---    _cache_tz = _get_localzone()
13:47:35---                ^^^^^^^^^^^^^^^^
13:47:35---  File "BirdNET-Pi/birdnet/lib/python3.11/site-packages/tzlocal/unix.py", line 180, in _get_localzone
13:47:35---    tzname = _get_localzone_name(_root)
13:47:35---             ^^^^^^^^^^^^^^^^^^^^^^^^^^
13:47:35---  File "BirdNET-Pi/birdnet/lib/python3.11/site-packages/tzlocal/unix.py", line 159, in _get_localzone_name
13:47:35---    raise zoneinfo.ZoneInfoNotFoundError(message)
13:47:35---zoneinfo._common.ZoneInfoNotFoundError: 'Multiple conflicting time zone configurations found:\n/etc/timezone: Europe/London\n/etc/localtime is a symlink to: America/Chicago\nFix the configuration, or set the time zone in a TZ environment variable.\n'

**Comment by Nachtzuster** [comment]
I think you ran into this one: https://github.com/Nachtzuster/BirdNET-Pi/issues/56

**Comment by Nachtzuster** [comment]
@dalogue1 I pushed a fix for this, could you give it a try?

**Reply by dalogue1** [reply]
How do I install the update?  Do I just reboot or do I need to run an update command from a command prompt or a menu choice?

**Reply by Nachtzuster** [reply]
in the menu: Tools->System Controls->Update

**Reply by dalogue1** [reply]
It works now.
